### PR TITLE
Form and form elements

### PR DESCRIPTION
**Pull request by vikispeaks**

### CODE REVIEW

File: building-web-forms/form-and-form-elements/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Script
+
+In this video, we will learn about HTML Forms and HTML elements used within it to collect user input in web pages.
```
Comment (avishekjana) [inline building-web-forms/form-and-form-elements/README.md:3]: we will learn about HTML Form and HTML elements used within it, to collect user input in web pages

File: building-web-forms/form-and-form-elements/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+The HTML `form` element is used to create an HTML form for user input. It is used as a container for elements such as text fields, checkboxes, radio buttons and submit buttons.
```
Comment (avishekjana) [inline building-web-forms/form-and-form-elements/README.md:5]: The HTML `form` element is used to create an HTML form for user input. 

- This sentence is almost identical to previous one.

File: building-web-forms/form-and-form-elements/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+Let's create a `form` that can collect user data. This element wraps all the other elements that go inside of our form. 
+
+> Action: Open the HTML Document created previously and add an Form element as below.
+
+```js
+<form>
+    ## form elements
+</form>
+```
+You cannot have forms nested inside another form. That means you cannot have a `form` element inside another `form` element.
```
Comment (avishekjana) [inline building-web-forms/form-and-form-elements/README.md:16]: That means, you cannot have a `form` element inside another `form` element.

File: building-web-forms/form-and-form-elements/README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+### Radio button
+
+The `input` element with type `radio` allows an user to select a single option from a limited number of choices.
+
+> Action: Open the HTML Document created previously and add a radio button to the Form element as below.
+
+### Submit button
+
+The `input` element with type `submit` allows an user to submit the collected user data to the web server.
```
Comment (avishekjana) [inline building-web-forms/form-and-form-elements/README.md:65]: The `input` element with type `submit` allows a user to send the form data to the web server.

File: building-web-forms/form-and-form-elements/README.md (hunk 2 above)
Comment (vikispeaks) [reply building-web-forms/form-and-form-elements/README.md:5]: This is just an explanation of the `form` tag itself. Keeping it as this is a reference to the element itself, unlike the previous text.

File: building-web-forms/form-and-form-elements/README.md (hunk 4 above)
Comment (vikispeaks) [reply building-web-forms/form-and-form-elements/README.md:65]: Updated.

**Comment by vikispeaks** [comment]
@mjzac @avishekjana Can one of you re-review and merge this.

**Review by mjzac — APPROVED** [review]

**Pull request merged**